docs/configuration.md

```diff
@@ -761,6 +761,17 @@ Apart from these, the following properties are also available, and may be useful
     Compression will use <code>spark.io.compression.codec</code>.
   </td>
 </tr>
+<tr>
+  <td><code>spark.eventLog.allowErasureCoding</code></td>
```

Comment:
Change `spark.eventLog.allowErasureCoding` to `spark.eventLog.allowErasureCoding.enabled` ?

Reply:
cc @Ngone51 Could you submit a PR?

Reply:
Sure.

Reply:
Added #26998